libenv/unix_iface.c

```diff
@@ -677,6 +676,57 @@ static void FindV6InterfacesInfo(EvalContext *ctx, Rlist **interfaces, Rlist **h
             }
         }
 
+
+        const char *const stripped_ifconfig_line =
+            TrimWhitespace(ifconfig_line);
+        if (StringStartsWith(stripped_ifconfig_line, "ether"))
+        {
+            Seq *const ether_line =
+                SeqStringFromString(stripped_ifconfig_line, ' ');
+            const size_t length = SeqLength(ether_line);
+
+            if (length < 2
+                || !StringSafeEqual_IgnoreCase(SeqAt(ether_line, 0), "ether"))
```

Comment:
If you changed the above `StringStartsWith()` to check for `ether ` (including the space), you'd be able to drop this second part of the condition here.